.github/bin/build-matrix-from-impacted.py

```diff
@@ -98,7 +98,7 @@ def build(matrix_file, impacted_file, output_file):
 def check_modules(modules, impacted):
     if isinstance(modules, str):
         modules = [modules]
-    if impacted and not any(module in impacted for module in modules):
```

Comment:
this was needed so that master branch builds run all the tests. now they will run none